src/pages/workspace/receiptPartners/EditInviteReceiptPartnerPolicyPage.tsx

```diff
@@ -0,0 +1,334 @@
+const UBER_EMPLOYEE_STATUS_VALUES = [
+    CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.LINKED,
+    CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.INVITED,
+    CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.LINKED_PENDING_APPROVAL,
+    CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.SUSPENDED,
+    CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.DELETED,
+    CONST.POLICY.RECEIPT_PARTNERS.UBER_EMPLOYEE_STATUS.NONE,
+] as const;
+
+type ReceiptPartnersTab = TupleToUnion<typeof TAB_NAMES>;
+type UberEmployeeStatus = TupleToUnion<typeof UBER_EMPLOYEE_STATUS_VALUES>;
+
+function EditInviteReceiptPartnerPolicyPage({route}: EditInviteReceiptPartnerPolicyPageProps) {
+    const styles = useThemeStyles();
+    const {translate, localeCompare} = useLocalize();
+    const {isOffline} = useNetwork();
+
+    const policyID = route.params.policyID;
+    const policy = usePolicy(policyID);
+
+    // Maintain independent search state per tab to avoid carryover across tabs
+    const [allSearchTerm, allDebouncedSearchTerm, setAllSearchTerm] = useDebouncedState('');
+    const [linkedSearchTerm, linkedDebouncedSearchTerm, setLinkedSearchTerm] = useDebouncedState('');
+    const [outstandingSearchTerm, outstandingDebouncedSearchTerm, setOutstandingSearchTerm] = useDebouncedState('');
+
+    const getSearchStateForTab = useCallback(
+        (tab: ReceiptPartnersTab) => {
+            if (tab === CONST.TAB.RECEIPT_PARTNERS.ALL) {
+                return {searchTerm: allSearchTerm, debouncedSearchTerm: allDebouncedSearchTerm, setSearchTerm: setAllSearchTerm};
+            }
+            if (tab === CONST.TAB.RECEIPT_PARTNERS.LINKED) {
+                return {searchTerm: linkedSearchTerm, debouncedSearchTerm: linkedDebouncedSearchTerm, setSearchTerm: setLinkedSearchTerm};
+            }
+            return {searchTerm: outstandingSearchTerm, debouncedSearchTerm: outstandingDebouncedSearchTerm, setSearchTerm: setOutstandingSearchTerm};
+        },
+        [
+            allSearchTerm,
+            allDebouncedSearchTerm,
+            linkedSearchTerm,
+            linkedDebouncedSearchTerm,
+            outstandingSearchTerm,
+            outstandingDebouncedSearchTerm,
+            setAllSearchTerm,
+            setLinkedSearchTerm,
+            setOutstandingSearchTerm,
+        ],
+    );
+
+    const uberEmployeesByEmail = useMemo<Record<string, {status?: string}>>(() => {
+        const policyWithEmployees = policy as typeof policy & {
+            receiptPartners?: {
+                uber?: {
+                    employees?: Record<string, {status?: string}>;
+                };
+            };
+        };
+        return policyWithEmployees?.receiptPartners?.uber?.employees ?? {};
+    }, [policy]);
+
+    const deriveStatus = useCallback(
+        (email: string): UberEmployeeStatus => {
+            const status = uberEmployeesByEmail[email]?.status?.toLowerCase();
```

Comment:
Why are you lower casing here and then below you can use the constants defined [here](https://github.com/Expensify/App/pull/68409/files#diff-ea78c710861a60147d2f3000e07ce41004df40cf732560c51cb1e714080ed69eR2990)

Reply:
Also seems the possible values are wrong (see comment on the constant I linked above)

Reply:
Why was this resolved?